src/libcommon/utility/types.h

```diff
@@ -369,7 +369,7 @@ enum class AppStateKey {
 };
 std::string toString(AppStateKey e);
 
-constexpr int64_t SELF_RESTARTE_DISABLE_VALUE = -1;
+constexpr int64_t SELF_RESTARTER_DISABLE_VALUE = -1;
```

Comment:
```suggestion
static constexpr int64_t SELF_RESTARTER_DISABLE_VALUE = -1;
```
You may check the second answer of https://stackoverflow.com/questions/13865842/does-static-constexpr-variable-inside-a-function-make-sense regarding the use if `static` in this context.

Reply:
Done